lib/hiptest-publisher/options_parser.rb

```diff
@@ -146,6 +146,7 @@ def self.all_options
       Option.new('l', 'language=LANG', 'ruby', String, "Target language", :language),
       Option.new('f', 'framework=FRAMEWORK', '', String, "Test framework to use", :framework),
       Option.new('o', 'output-directory=PATH', '.', String, "Output directory", :output_directory),
+      Option.new(nil, 'filename-pattern=PATTERN', nil, String, "Filename pattern (containing %s)", :filename_pattern),
```

Comment:
Don't let security experts see this. Substitution patterns provided by the user. :wink: